### ISSUE TITLE
Loss function in BackpropTrainer

### ISSUE THREAD

**Issue by mfornet** [pybrain/pybrain#231]
Is there any way to change the loss function when we are applying backpropagation to train the network? I want to use cross entropy (CE) instead of mean squared error (MSE) which is the default.

I have read this issue [here](https://github.com/pybrain/pybrain/issues/20#issuecomment-46452172), but I see no explicit answer, and in my opinion it is very important to have at least most common loss functions implemented. There is a way to [do it](https://github.com/pybrain/pybrain/issues/20#issuecomment-269718464), but it involves changing source code of pybrain.
